### ISSUE TITLE
Order of snapshots within ini changing

### ISSUE THREAD

**Issue by cser** [guilryder/clavier-plus#28]
Every time after restart Clavier file ini is changed.
It's not a big problem, but it's hard to understand what's changed by me and what's changed by Clavier after synchronization.
May be sort snapshots before write them into Clavier.ini?

Clavier.ini at first start
```
Language=English
Size=637,495,0,0
Columns=35,20,15,10
Sorting=0

Shortcut=Ctrl + Shift + T
Code=1620
Text=[{Wait,50}][{KeysDown, Ctrl}][{Wait,50}][{KeysDown}][Ctrl + A][Ctrl + X][{Wait,50}][[c:\\methods\\Apps\\cli\\nircmd.exe exec hide "c:\\methods\\Apps\\cli\\sed-imath-explain.bat"]]
Usages=31
-
Shortcut=Ctrl + 4
Code=564
Text=\[\][Left]
AllProgramsBut=notepad3.exe
Usages=3
-
Shortcut=Ctrl + 5
Code=565
Text=\{\}[Left]
AllProgramsBut=notepad3.exe
Usages=3
-
Shortcut=Ctrl + 6
Code=566
Text=``[Left]
AllProgramsBut=notepad3.exe
Usages=3
-
```
Clavier.ini at second start
```
Language=English
Size=637,495,0,0
Columns=35,20,15,10
Sorting=0

Shortcut=Ctrl + 6
Code=566
Text=``[Left]
AllProgramsBut=notepad3.exe
Usages=3
-
Shortcut=Ctrl + 5
Code=565
Text=\{\}[Left]
AllProgramsBut=notepad3.exe
Usages=3
-
Shortcut=Ctrl + 4
Code=564
Text=\[\][Left]
AllProgramsBut=notepad3.exe
Usages=3
-
Shortcut=Ctrl + Shift + T
Code=1620
Text=[{Wait,50}][{KeysDown, Ctrl}][{Wait,50}][{KeysDown}][Ctrl + A][Ctrl + X][{Wait,50}][[c:\\methods\\Apps\\cli\\nircmd.exe exec hide "c:\\methods\\Apps\\cli\\sed-imath-explain.bat"]]
Usages=31
-
```

**Comment by cser** [comment]
OK.

> May be sort snapshots before write them into Clavier.ini?

It was a bad idea - may be some UI commands to change order will be added in the future.
It's would be better to try to save list order for all operations.

**Comment by guilryder** [comment]
> may be some UI commands to change order will be added in the future.

The UI already allows this: you can sort the list by clicking the column headers.

> It's would be better to try to save list order for all operations.

Yes, that's what I ended up doing. I also reduce the number of times Clavier+ saves its configuration: read-only operations such as loading a configuration file at startup don't need to save anything. These changes seem to work fine, but I need to do more testing for other changes made since the last release version. Beta version in the mean time: http://utilfr42.free.fr/dn/beta/Clavier.exe

**Comment by cser** [comment]
There have been no changes in the order yet since new exe using

**Comment by guilryder** [comment]
Released in Clavier+ 11.2.1.
